devtools/common/src/main/java/io/quarkus/templates/rest/BasicRest.java

```diff
@@ -28,9 +25,9 @@
 
     private Map<String, Object> context;
     private String path = "/hello";
-    private File projectRoot;
-    private File srcMain;
-    private File testMain;
+    private Writer writer;
+    private String srcMain;
```

Comment:
With the changes being made, perhaps now `srcMain` and `testMain` should be renamed to `srcMain` and `testMain` to make their intent clear (since readers can no longer count on the type - just `String` now)

Reply:
rename srcMain to srcMain... heu...how to say that.... it's exactly the same name. ;o)
I propose srcMainPath ? and testMainPath too

Reply:
LOL - Yeah that's what I meant :P